### ISSUE TITLE
Documentation wanted for three different per-user resource controls

### ISSUE THREAD

**Issue by ghost** [systemd/systemd#9590]
**Is your feature request related to a problem? Please describe.**

Here I would like to request for two enhancements in documentation.

1. `UserTasksMax` in `logind.conf` was replaced by `TasksMax` in `/usr/lib/systemd/system/user-.slice.d/10-defaults.conf` in [this commit](https://github.com/systemd/systemd/commit/284149392755f086d0a714071c33aa609e61505e#diff-62ae4f4af4b3a323896b6bf7bdd27ab7) and its explanation was also removed from [logind.conf(5) manpage](https://www.freedesktop.org/software/systemd/man/logind.conf.html). Configuration of per-user `TasksMax` is suggested in the above commit as 

```
+        log_syntax(unit, LOG_NOTICE, filename, line, 0,
+                   "Support for option %s= has been removed.",
+                   lvalue);
+        log_info("Hint: try creating /etc/systemd/system/user-.slice/50-limits.conf with:\n"
+                 "        [Slice]\n"
+                 "        TasksMax=%s",
+                 rvalue);
```

(I think `user-.slice` should be `user-.slice.d` in the above change.)

But usage of `/etc/systemd/system/user-.slice` is not explained (I am sorry if I overlooked one).

2. New feature `systemd.tasks_max` was introduced and explained in [pam_systemd(8) manpage](https://www.freedesktop.org/software/systemd/man/pam_systemd.html#systemd.tasks_max). In addition to the above two kinds of per-user `TasksMax`, one can set `TasksMax` also in `/etc/systemd/system/user@.conf.d/`. Some explanations about `user-.slice.d/*.conf` and `user@.conf.d/*.conf` seem desirable, and an explanation of the difference among these three `TasksMax` is probably helpful.

**Describe the solution you'd like**

* Explanations of `user-.slice.d/*.conf` and `user@.conf.d/*.conf`.
* Explanation of the difference among the above two and `systemd.tasks_max`

**Describe alternatives you've considered**

* I am not sure where the explanations should be placed. One option could be [systemd.resouce-control(5) manpage](https://www.freedesktop.org/software/systemd/man/systemd.resource-control.html). 

* Explanation of `user-.slice.d/*.conf` could be

> All the processes of an ordinary user $UID are placed under `user-$UID.slice`. All resource control directives for slices in `systemd.resource-control(5)` can be used in system administrator's configurations of `user-$UID.slice`, which are typically placed in `/etc/systemd/system/user-.slice.d/*.conf` files. Those resource controls are applied to the total resource consumptions by all the processes with $UID.

* I do not clearly know what is `user@.conf.d/*.conf`, nor the difference among the three per-user resource controls... I wonder if `Delegate=cpu` is required for using `-p CPUQuota=100` and `-p CPUAccounting=yes` with `systemd-run --user`. Sessions and scopes started by `systemd-run --user` are placed under `user@$UID.service` cgroup and resource controls in `user@.conf.d/*.conf` apply to `systemd-run --user`. So if a scope started by `systemd-run --user --scope` wants CPU delegation, then `Delegate=cpu` is probably required in `user@$UID.service`. I would like to see explanations on such topics.

**Comment by poettering** [comment]
Hmm, the path in the log message is indeed borked. We should fix that.

And yeah, we should certainly document this better.

cc @keszybz

**Comment by yuwata** [comment]
The log message is fixed by #9601.
